lib/ddtrace/contrib/redis/patcher.rb

```diff
@@ -59,7 +59,7 @@ def patch_redis_client
             end
 
             def initialize(*args)
-              pin = Datadog::Pin.new(SERVICE, app: 'redis', app_type: Datadog::Ext::AppTypes::CACHE)
+              pin = Datadog::Pin.new(SERVICE, app: 'redis', app_type: Datadog::Ext::AppTypes::DB)
```

Comment:
Our "service type" still lack a real Spec, but for now let's just have it consistent with Python. So here, gtm!